packages/trees/frontend/sirius-components-trees/src/treeitems/TreeItem.tsx

```diff
@@ -291,14 +269,13 @@ export const TreeItem = ({
   };
 
   const dragStart: React.DragEventHandler<HTMLDivElement> = (event) => {
-    const isDraggedItemSelected = selection.entries.map((entry) => entry.id).includes(item.id);
+    const isDraggedItemSelected = selectedTreeItemIds.map((id) => id).includes(item.id);
     if (!isDraggedItemSelected) {
       // If we're dragging a non-selected item, drag it alone
-      const itemEntry: SelectionEntry = { id: item.id, kind: item.kind };
-      event.dataTransfer.setData(DRAG_SOURCES_TYPE, JSON.stringify([itemEntry]));
-    } else if (selection.entries.length > 0) {
+      event.dataTransfer.setData(DRAG_SOURCES_TYPE, JSON.stringify([item.id]));
+    } else if (selectedTreeItemIds.length > 0) {
       // Otherwise drag the whole selection
-      event.dataTransfer.setData(DRAG_SOURCES_TYPE, JSON.stringify(selection.entries));
+      event.dataTransfer.setData(DRAG_SOURCES_TYPE, JSON.stringify(selectedTreeItemIds));
```

Comment:
We will need to document that the value of the data transfer has been changed from `{id: string}[]` to `string[]`

Reply:
Done